.eslintrc.json

```diff
@@ -8,6 +8,23 @@
 		"curly": "off",
 		"indent": "off",
 		"no-confusing-arrow": "off",
-		"no-mixed-operators": "off"
+		"no-mixed-operators": "off",
+		"import/no-extraneous-dependencies": [
+			"error", {
+				"devDependencies": true,
+				"packageDir": "./"
+			}
+		]
+	},
+	"settings": {
+		"import/resolver": {
+			"node": {
```

Comment:
Is this necessary?

Reply:
eslint also needs to know how to resolve the alias